### DISCUSSION TITLE
I can't shift + right click

### DISCUSSION THREAD

**Discussion by ozi2285** [ORelio/Minecraft-Console-Client#2571] [Q&A]
in guide they say there is 2 version of click but in client not same way they explain they say exp: /inventory container (click|shiftclick) 14 rightclick but in client it self it's like this /inventory container click 14 shiftclick which won't right click the one i need help

**Comment by ozi2285** [comment]
why there is o answer at all :(

**Reply by milutinke** [reply]
Maybe because you haven't phrased your question correctly, no one can understand what are you trying to ask.
 Re-phrase your question so it's understandable.
 
 If I understood it, you're asking for a shift + right click?
 The `Shift + Right Click` in Minecraft is the same as `Shift + Left Click`, it does the same action, so if you need that, use ShiftClick option.
 If you only need right click, use `right` as an option.
